src/index.js

```diff
@@ -10,4 +10,12 @@ Reflux.listenTo = require('./listenTo');
 
 Reflux.listenToMany = require('./listenToMany');
 
+/* globals React: false */
+Reflux.defineReact = require('./defineReact');
+try {
+	if (React) {
+		Reflux.defineReact(React, Reflux);
+	}
+} catch (e) { }
```

Comment:
Great idea, I love it. It's implemented, passes all tests, and is now pushed to the fork/pull request.